src/action/updateRoleEnvironmentAccess.ts

```diff
@@ -8,6 +8,13 @@ export default async function ({ space, environment }: { space: Space; environme
   try {
     Logger.log(`Updating the Role ${roleId} to have access on Environment: ${environment.sys.id}.`);
     const role = await space.getRole(roleId);
+    const isEnvironmentIdInRolePolicy = role.policies.some((policy) =>
+      containsEnvironmentId(policy.constraint, environment.sys.id),
+    );
+    if (isEnvironmentIdInRolePolicy) {
+      Logger.log(`The Role ${roleId} has already been given access to Environment ${environment.sys.id}.`);
+      return; // eslint-disable-line padding-line-between-statements
```

Comment:
Why disable? Let’s satisfy the linter and add that blank line before the return.

Reply:
adding the blank line initially gave me an prettier lint error:
![image](https://github.com/user-attachments/assets/555285d4-8346-45fe-8642-b0608678cad0)

but if i backspace several times on that blank line, it fixes what i thought was conflicting lint issues.